recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
   # variant is slightly preferred by conda's solver.
   number: {{ number + 100 }}  # [variant=="plugin"]
   number: {{ number }}        # [variant=="patch"]
-  skip: True # [py<=36]
+  skip: True # [py<=36 or variant=="patch" and py>311]
```

Comment:
The versions of conda that we are making available for Python 3.12 and later are not old enough to utilize the `patch` version of the package.